webpack/assets/javascripts/react_app/components/user/passwordStrength/passwordStrength.test.js

```diff
@@ -0,0 +1,62 @@
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import PasswordStrength from './';
+import {
+  initialState,
+  passwordMatch,
+  passwordNotMatched,
+} from './passwordStrength.fixtures';
+
+configure({ adapter: new Adapter() });
+
+jest.unmock('./');
+global.__ = jest.fn();
+const mockStore = configureMockStore([thunk]);
+
+function setup(verify, error, state) {
+  const store = mockStore(state);
+
+  return mount(<PasswordStrength
```

Comment:
would shallow work here too?

Reply:
shallow doesn't render inner components therefore the resulted snapshot lacks in inner content